Examples/CrossBreeze_CrossTest_ExampleTests/App.config

```diff
@@ -144,7 +144,7 @@
 	<!-- The connection strings. -->
 	<connectionStrings>
 		<!-- Define the database servers to use. The name is the name of the server and the connection string should be without a database. -->
-		<add name="ExampleMsSqlConnection" connectionString="Data Source=localhost;User Id=sa;Password=Test1234;" />
-		<add name="ExampleMsSsasConnection" connectionString="Data Source=localhost\LocalDev2017;" />
+		<add name="ExampleMsSqlConnection" connectionString="Data Source=localhost\LocalDev2017;Integrated Security=SSPI;"/>

```

Comment:
You revert back to a local SQL Server i.s.o. a docker container? Does the docker version not support snapshot isolation?

Reply:
That was unintentional, it also works in the docker version, also modified the SQL script to allow snapshot isolation on the example database